src/IgbinarySerializer.php

```diff
@@ -1,18 +1,33 @@
 <?php
 
+declare(strict_types=1);
+
 namespace Yiisoft\Serializer;
 
+use function igbinary_serialize;
+use function igbinary_unserialize;
+
 /**
  * IgbinarySerializer uses [Igbinary PHP extension](http://pecl.php.net/package/igbinary) for serialization.
- * Make sure you have 'igbinary' PHP extension install at your system before attempt to use this serializer.
+ * Make sure you have "igbinary" PHP extension install at your system before attempt to use this serializer.
  */
 final class IgbinarySerializer implements SerializerInterface
 {
+    /**
+     * {@inheritDoc}
```

Comment:
This whole docblock doesn't add anything:

1. It implements an interface which has description already.
2. A referece to `igbinary_serialize()` is obvous from the code.

I'd keep phpdoc in this case clean.